src/__tests__/test-mock-rpc-handlers.browser.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+import test from 'tape-cup';
+import sinon from 'sinon';
+import getMockRpcHandlers from '../mock-rpc-handlers';
+import ResponseError from '../response-error';
+
+test('mockRpcHandlers', async t => {
+  t.plan(4);
+  const rpcFixtures = {
+    GET_USER_SUCCESS: {
+      getUser: {
+        firstName: 'John',
+        lastName: 'Doe',
+        uuid: 123,
+      },
+    },
+    UPDATE_USER: {
+      updateUser: [
+        {
+          args: [{firstName: 'Jane'}],
+          response: {
+            firstName: 'Jane',
+            lastName: 'Doe',
+            uuid: 123,
+          },
+        },
+        {
+          args: [{firstName: ''}],
+          response: new ResponseError('Username cant be empty'),
+        },
+      ],
+    },
+  };
+
+  const onMockRpcSpy = sinon.spy();
```

Comment:
Can we update this to use `jest.fn()` since `fusion-cli` ships with jest

Reply:
will do

Reply:
sorry, I was confused. Ignore this

Reply:
i am getting ReferenceError: jest is not defined when i used `jest.fn`
in fusion apps the jest global was injected via `fusion-cli`